metapackages/auto-instrumentations-node/src/register.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as opentelemetry from '@opentelemetry/sdk-node';
+import { diag, DiagConsoleLogger } from '@opentelemetry/api';
+import {
+  getNodeAutoInstrumentations,
+  getResourceDetectorsFromEnv,
+} from './utils';
+
+diag.setLogger(
+  new DiagConsoleLogger(),
+  opentelemetry.core.getEnv().OTEL_LOG_LEVEL
+);
+
+const sdk = new opentelemetry.NodeSDK({
+  instrumentations: [getNodeAutoInstrumentations()],
+  resourceDetectors: getResourceDetectorsFromEnv(),
+});
+
+try {
+  sdk.start();
+  diag.debug('SDK initialized');
+} catch (error) {
+  diag.error('Error initializing SDK', error);
```

Comment:
I would also add here that it's an **OpenTelemetry** SDK, and a sentence saying something like "Your application is not instrumented and will not produce telemetry."